MaybeGame/Player.cs

```diff
@@ -16,19 +16,23 @@ public class Player
         public double DodgeChance { get; set; }
         public double Speed { get; set; }
         public int AttackPower { get; set; }
+        public double ExperiencePoints { get; set; }
+        public double SavedExperiencePoints { get; set; }
```

Comment:
Why do you need **SavedExperiencePoints** here?

Reply:
This will be used later to know how much Experience point has each player collected throughout the game, since right now the only way to know the level is by running LevelUp method on loading. Willl probobly have a different method to do this later so it should be commented since it's not being used.